internal/cmd/skupper/site/kube/site_update.go

```diff
@@ -236,7 +236,7 @@ func (cmd *CmdSiteUpdate) WaitUntilReady() error {
 			return err
 		}
 
-		if resource != nil && resource.Status.StatusMessage == "OK" {
+		if resource != nil && resource.IsActive() {
```

Comment:
In pkg/apis/skupper/v1alpha1/types.go Should we change IsActive to IsConfigured for the Site?

Reply:
I agree, that would be clearer. (Bad naming on my part!)

Reply:
fixed: a958b42f8caa260a955778454ed2b4404992512c